### DISCUSSION TITLE
Design Tip: Knight Lab's TimelineJS Works Well

### DISCUSSION THREAD

**Discussion by writingpoli** [knowledgefutures/pubpub#1626] [Show and tell]
Using iframe embed, it works quite well: 

[Interactive Timeline of Irresponsibility](https://www.sareview.org/pub/patetimeline)

**Comment by gabestein** [comment]
👍 for all of Knight Lab's tools. We also used their map tool, here: https://tmb.apaopen.org/pub/agy1p3cr/release/6

**Comment by jarmitage** [comment]
These look fantastic. Is their full width capability a custom integration, or can any iframe do that?

**Reply by writingpoli** [reply]
You should be able to control it under the Media tab after embedding your iframe.

**Reply by jarmitage** [reply]
Got it! Thank you
